src/propTypes/HandlePropTypes.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+
+export default {
+  'aria-valuemax': PropTypes.number,
+  'aria-valuemin': PropTypes.number,
+  'aria-valuenow': PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  'aria-disabled': PropTypes.bool,
+  'data-handle-key': PropTypes.node,
+  orientation: PropTypes.string,
+  disabled: PropTypes.bool,
+  onClick: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  onMouseDown: PropTypes.func,
+  onTouchStart: PropTypes.func,
+  handleRef: PropTypes.func,
+  role: PropTypes.string,
+  style: PropTypes.object,
+  tabIndex: PropTypes.number,
```

Comment:
i think this needs to be an integer, -1, or >= 0?

Reply:
integer >= -1

Reply:
either -1 or 0, anything greater than 0 can lead to unexpected tab orders

Reply:
in that case, `PropTypes.oneOf([-1, 0])` seems useful :-)